cmd/flags.go

```diff
@@ -127,6 +127,11 @@ func storageFlags() []cli.Flag {
 			Value: 20,
 			Usage: "number of connections to upload",
 		},
+		&cli.IntFlag{
+			Name:  "max-stage-write",
+			Value: 0, // Enable this to have concurrent uploads to two backends, and get write bandwidth equals to sum of the two
```

Comment:
Maybe we can enable it by default, with a value 10 or 20.

Reply:
I'm afraid enabling this by default will break current expectation for `write-back`, is that a concern to you?

Also, it's hard to set a default value without observing `staging_writing_blocks` metric first. User may have multiple disks for staging, or only have HDD for staging, etc. Different environments require different optimal value.